IceCream/Classes/SyncObject.swift

```diff
@@ -175,10 +178,16 @@ extension SyncObject: Syncable {
         }
     }
     
-    public func pushLocalObjectsToCloudKit() {
+    public func deleteCloudKitRecords(completion: ((Error?) -> ())?) {
```

Comment:
since this completion is optional, should we pass in `nil` as default ?

Reply:
@jacobbullock sure, done!